added localization id.json

febryardiansyah opened the pull request.

SincerelyFaust commented:
Hi, we use the translation service GitLocalize. It is way more convenient and easy. Maybe you can copy your translations over to it.
https://gitlocalize.com/repo/5170

febryardiansyah commented:
hello, thanks for the response, i've updated via GitLocalize. You can check mine here https://gitlocalize.com/repo/5222
And you can check last commit i added
